controller/internal/enforcer/proxy/enforcerproxy.go

```diff
@@ -68,6 +68,10 @@ func (s *ProxyInfo) InitRemoteEnforcer(contextID string) error {
 		return fmt.Errorf("failed to initialize remote enforcer: status: %s: %s", resp.Status, err)
 	}
 
+	if resp.Status != "" {
```

Comment:
We were only checkin that the RPC return was errored and only then the status. 
Now if the status is not empty, we display a warning, but I think this should always result in an error